starboard/android/shared/player_components_factory.h

```diff
@@ -97,7 +97,7 @@ class AudioRendererSinkAndroid : public ::starboard::shared::starboard::player::
                   audio_frame_storage_type, frame_buffers,
                   frame_buffers_size_in_frames, update_source_status_func,
                   consume_frames_func, error_func, start_media_time,
-                  tunnel_mode_audio_session_id, enable_audio_device_callback,
+                  tunnel_mode_audio_session_id, true,
```

Comment:
You can go further and remove this parameter entirely. This makes the change a little more complex as you'll need to modify at least audio_track_audio_sink_type.* and AudioOutputManager.java, but the end result will be simpler.

Reply:
Cleanup the parameter entirely.